lib/polymorphic_integer_type/mapping.rb

```diff
@@ -15,6 +15,6 @@ def self.[](as)
       @@mapping[as] || {}
     end
 
+    singleton_class.send(:alias_method, :[]=, :add)
```

Comment:
thought it was odd we have an `#add` method that behaves canonically like `[]=` so made an alias 🤷‍♂️